system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -209,11 +215,11 @@ parameter_types! {
 	pub XcmAssetFeesReceiver: Option<AccountId> = Authorship::author();
 }
 
-match_types! {
-	pub type ParentOrParentsPlurality: impl Contains<MultiLocation> = {
-		MultiLocation { parents: 1, interior: Here } |
-		MultiLocation { parents: 1, interior: X1(Plurality { .. }) }
-	};
+pub struct ParentOrParentsPlurality;
+impl Contains<Location> for ParentOrParentsPlurality {
+	fn contains(location: &Location) -> bool {
+		matches!(location.unpack(), (1, []) | (1, [Plurality { .. }]))
+	}
```

Comment:
Do we have end-to-end tests to know that referenda passed on the Relay Chain / Collectives will match correctly here and execute the privileged logic? Would be good to test some scenarios on Zombienet or Chopsticks (@NachoPal )

Reply:
We have e2e tests for cases: OpenGov from Relay Chain executes some permissioned operation on AssetHub. OpenGov inits Fellowship on Collectives, Fellowship whitelists the call, Fellowship executes some permissioned operation on AssetHub. Not sure where these tests located now, but we had them in polkadot-sdk before.